participants/BT2Lotus.md

```diff
@@ -0,0 +1,61 @@
+# BT2Lotus
+
+## (0) Registration Submission, Project Name, and Team Slack Handles
+
+*Submission ID: 674109622
+Project Name: BT2Lotus
+Slack Name: thick forest
+MemberID: U01B7F9NE22*
+
+## (1) Brief Project Description
+
+*Sync BitTorrent files to lotus*
+
+## (2) Link to Project UI
+
+*[BT2Lotus Home Page](http://bt2lotus.com)*
+
+## (3) What does your application/UI do?
+
+*Users can submit BitTorrent file(*.torrent) or magnet link(magnet:*), then we save the data to the lotus network.*
```

Comment:
Please ensure that the torrents stored are legal and have an appropriate license.